### ISSUE TITLE
[Screen Reader – Intelligent Support - IRIS]: The error message that an incorrect alias has been selected is not being announced by the screen reader

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4612]
### **PWD Impact:**
Users relying on screen reader will be affected if the errors that appear on the page are not being announced.

### **Repro Steps:**
Step 1: Open the above URL in a browser. Home Page will open. 
Step 2: Navigate to Iris bot icon in the header and invoke it
Step 3: Iris bot chat window is displayed
Step 4: Type start over-> Top questions for my role-> select Where can I download the Time 2.0 mobile app for Android ->Select no create a support ticket->select any tool from the drop down->continue button
Step 5: After invoking the continue button a form is displayed, turn on Narrator.
Step 6: Enter into the 'Alias of user/groups to receive all incident emails' editable field and here enter an incorrect alias.
Step 7: Verify if the narrator announces the error that appears.
### **Actual result:**
When we enter an incorrect alias an error message appears 'One or more of the email aliases are invalid. The following are incorrect: abcdef. Please correct the alias(es) to continue creating your incident.'
Now this error is not announced by the screen reader, instead it announces a lot of unwarranted changes.
### **Expected Result:**
If the alias that has entered is incorrect and an error message pops up at one point the narrator should announce that an error has occurred. There should be atleast one announcement that should mention the field where the error occurred and what the error message was.

### **Note:** 
The behavior exhibited by NVDA is also similar.

### **MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b49C317E6-0FEE-4B0A-9055-7A914300DAEF%7d&file=MAS%203.3.1%20%E2%80%93%20Error%20Identification.docx&action=default&mobileredirect=true
### Test Environment:
OS: Windows 11 Enterprise version 21H2
Browser: New Edge Version 106.0.1370.42 (Official build) (64-bit)
URL: https://servicesuat.microsoft.com/#/
Screen Reader: Narrator

**Comment by pgopinadhu** [comment]
#A11yMAS;#A11ySev2;#HCL;#MAS3.3.1;#Accessibility;#Win11-Edge(Chromium);#BotFrameworkWebChat-Jul22;

**Comment by compulim** [comment]
@pgopinadhu I couldn't see the bot using the URL with my internal credentials.

Could you grant me access to the Iris bot? We need access to understand how this issue repro, if this repro outside of the hosting page, and if any customizations were applied.

**Comment by compulim** [comment]
I have been asking access for the few weeks and still did not get a response.

We cannot investigate any further whether this is an issue with Web Chat or Adaptive Cards. I am closing this.

**Comment by pgopinadhu** [comment]
@compulim Product Team Provided the Environment access. Please check and let us know if there are any issues. Thanks!

**Comment by compulim** [comment]
No repro.

After navigating to the form, I can hear "Alias of users slash groups to receive all incident emails. Edit. Separate each user slash group with a comma or semicolon. Space"

This starts around 00:22 in the attachment.

https://user-images.githubusercontent.com/1622400/224420192-25c523d9-2777-4f1f-9515-0521055a7c55.mp4
